.gitignore

```diff
@@ -19,3 +19,6 @@ electron/pub
 /src/component-index.js
 /.tmp
 /webpack-stats.json
+/pegacorn-communicate-matrix-js-sdk
+/pegacorn-communicate-matrix-react-sdk
```

Comment:
I agree, certificate and temporary directories in app-web needs to be ignored after docker build process . I had done this as part of very first commit on feature/dockerise-lingo-app but later that changed as multiple devs worked in that branch.